src/mm-input/mm-input.js

```diff
@@ -34,6 +34,11 @@
 				value: false,
 				reflectToAttribute: true
 			},
+			readonly: {
```

Comment:
yeah i guess that makes sense. frustrating that its inconsistent with literally all of the other 2 word props but we should definitely follow native where we can